core/src/main/java/com/linecorp/armeria/common/AbstractAggregatedHttpMessage.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2019 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.common;
+
+abstract class AbstractAggregatedHttpMessage implements AggregatedHttpMessage {
+
+    private final HttpData content;
+    private final HttpHeaders trailingHeaders;
+
+    AbstractAggregatedHttpMessage(HttpData content, HttpHeaders trailingHeaders) {
+        this.content = content;
+        this.trailingHeaders = trailingHeaders;
+    }
+
+    @Override
+    public HttpData content() {
+        return content;
+    }
+
+    @Override
+    public HttpHeaders trailingHeaders() {
```

Comment:
This is probably a good time to deprecate `trailingHeaders` in favor of `trailers`, which we have consolidated on in other APIs (e.g., `additionalResponseTrailer`). What do you think? If so, let's rename it in all the parameters throughout the PR too.

Reply:
I prefer `trailers` to `trailingHeaders`. :-) If there's no one who agin this, I will rename it.

Reply:
:heavy_plus_sign: :one: